src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveController.kt

```diff
@@ -63,6 +64,11 @@ class OppgaveController(private val oppgaveService: OppgaveService) {
         return success(oppgaveService.finnMapper(finnMappeRequest))
     }
 
+    @GetMapping(path = ["/mappe/finn/{enhetNr}"])
+    fun finnMapper(@PathVariable enhetNr: String): Ressurs<List<MappeDto>> {
```

Comment:
Hvorfor ikke bruke den endepunkt som finnes å sende med FinnMappeRequest?

Reply:
Kan den være sånn att det finnes behov for å ta i bruk tema eller noen av de andre feltene? 
Føler det blir unødvendig vedlikehold hvis vi skal ha flere endepunkter som gjør den samme saken, og då burde kanskje den andre markeres som DEPRECATED og bruk av den burde fjernes? 

Husk att det kan være fint å ha med behover for endringer i commit msg

Reply:
Jeg skal prøve å bedre commit-meldingene mine. Jeg tror det bare er EF som bruker endepunktet i dag, så jeg har lyst til å deprecate det gamle.